client/state/themes/reducer.js

```diff
@@ -19,18 +19,70 @@ import {
 	THEMES_REQUEST,
 	THEMES_REQUEST_SUCCESS,
 	THEMES_REQUEST_FAILURE,
+	THEME_ACTIVATE_REQUEST_SUCCESS,
+	ACTIVE_THEME_REQUEST,
+	ACTIVE_THEME_REQUEST_SUCCESS,
+	ACTIVE_THEME_REQUEST_FAILURE,
 	SERIALIZE,
 	DESERIALIZE
 } from 'state/action-types';
 import {
 	getSerializedThemesQuery
 } from './utils';
 import { createReducer, isValidStateWithSchema } from 'state/utils';
-import { queriesSchema } from './schema';
+import { queriesSchema, activeThemesSchema } from './schema';
 import currentTheme from './current-theme/reducer';
 import themesUI from './themes-ui/reducer';
 
 /**
+ * Returns the updated theme active theme request state after an action has been
```

Comment:
Wording/typo -- should probably be `Returns the updated active theme state [...]`